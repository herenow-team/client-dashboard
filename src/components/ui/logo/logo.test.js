import React from 'react'
import renderer from 'react-test-renderer'
import Logo from '.'

describe('<NavbarItem /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Logo />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
