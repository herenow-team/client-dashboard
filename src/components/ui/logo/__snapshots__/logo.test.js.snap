// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarItem /> component should render correctly 1`] = `
<img
  alt="herenow"
  className="logo"
  src="nothing"
/>
`;
